docs/components.md

```diff
@@ -57,6 +57,39 @@ Setting `exposedEvents` to `true` rather than an array will expose all events.
 
 Additional events can also be exposed by passing in an array of event names to `exposedEvents` on the component options object.
 
+Events on components are namespaced when exposed. This means the events are bound to only the model of the component regardless of what level in the applicaton it is being used.
+
+To respond to events on a component's model, a parent or ancestor application/component can listen to the event on the model but will have to scope the listening action to the component. Events on the component's model must be exposed to its ancestor via the `exposedEvents` key.
+
+```javascript
+var ClockModel = BaseModel.extend({
+  postInitialize: function() {
+    this.listenTo(this, 'change:time_component:est_time', this.updateTime);
+  },
+
+  updateTime: function() {
+    // update the time
+  }
+});
+```
+
+Shown below is how the `change` event for `est_time` and a custom `tick` events on the `time_component` is exposed.
+
+var TimeModel = BaseModel.extend({
+  exposedEvents: ['change:est_time', 'tick:time_component'],
```

Comment:
Shouldn't this just expose `tick`?  `tick:time_component` isn't working for me, which makes sense because it shouldn't be concerned with its parent prop name (i.e., time_component).